koap/build.gradle.kts

```diff
@@ -52,4 +52,11 @@ npmPublishing {
             registry = uri("https://npm.pkg.github.com")
         }
     }
+
+    publications {
+        val js by getting {
+            // Workaround for bug https://github.com/npm/cli/issues/2143
+            shrinkwrapBundledDependencies = true
```

Comment:
FYI: This is enabled by default

Reply:
Cool, did not realize this was the case. Removed.